src/loaders/OmeZarrLoader.ts

```diff
@@ -358,11 +359,14 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
     if (!this.maxExtent) {
       this.maxExtent = loadSpec.subregion.clone();
     }
-    const pxDims0 = convertSubregionToPixels(
-      loadSpec.subregion,
-      new Vector3(shape0[x], shape0[y], hasZ ? shape0[z] : 1)
-    );
-    const pxSize0 = pxDims0.getSize(new Vector3());
+    // TODO IS IT SAFE TO IGNORE THIS SUBREGION STUFF?
```

Comment:
probably not ok but need to figure out where to store the subregion-scaled volume size